requirements.txt

```diff
@@ -1,4 +1,4 @@
-bittensor
+bittensor==7.0.1
```

Comment:
In our testing for finetuning we struggled with anything > 6.9.3, so I'd recommend caution with this. Pinning is a good idea, but consider 6.9.3 (when we last checked, almost no other subnet used > 6.9.3). In particular, logging seems to be broken

Reply:
I need to look into that yes. Haven't tested with this version yet.